src/test/java/MainTableSort.java

```diff
@@ -0,0 +1,61 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class MainTableSort {
+    private static ChromeDriver driver;
+
+    public static void main(String[] args) {
+        driver = new ChromeDriver();
+        driver.get("https://www.seleniumeasy.com/test/table-sort-search-demo.html");
+
+        List<Employee> employeesFromAutoDepartment = getEmployees(20, 40000);
+        List<Employee> employeesFromDevopsDepartment = getEmployees(40, 100000);
+
+        driver.quit();
+    }
+
+    public static List<Employee> getEmployees(int ageMin, int salaryMax) {
+        ArrayList<Employee> employees = new ArrayList<Employee>();
+        Select dropdown = new Select(driver.findElement(By.cssSelector("label > select")));
+        dropdown.selectByVisibleText("100");
```

Comment:
Let's change to "Show 10 entries"
Then, you will need to change your code to make sure that it works as expected